PrivateCollections/Ethiopia/MentasnotAbbuhay/MEAG001.xml

```diff
@@ -0,0 +1,497 @@
+                                 <dim unit="leaf">10</dim>
+                                 <locus from="159r" to="168v"/>
+                              </item>
+                              <item xml:id="q18" n="18">
+                                 <dim unit="leaf">10</dim>
+                                 <locus from="169r" to="178v"/>
+                              </item>
+                              <item xml:id="q19" n="19">
+                                 <dim unit="leaf">8</dim>
+                                 <locus from="179r" to="186v"/>
+                              </item>
+                              <item xml:id="q20" n="20">
+                                 <dim unit="leaf">8</dim>
+                                 <locus from="187r" to="194v"/>
+                              </item>
+                              <item xml:id="q21" n="21">
+                                 <dim unit="leaf">4</dim>
+                                 <locus from="195r" to="198v"/>
+                              </item>
+                           </list>
+                        </collation>
+                        <condition key="good"/>
+                     </supportDesc>
+                  </objectDesc>
+                             
+                                               
+                  <bindingDesc>
+                     <binding contemporary="true" xml:id="binding">
+                        <decoNote xml:id="b1">Blind-tooled brown leather cover.</decoNote>
+                        <decoNote xml:id="b2" type="Boards">Wooden boards with multicoloured <term key="textileInlay">textile inlays</term></decoNote>
+                        <decoNote xml:id="b3" type="bindingMaterial">
+                           <material key="wood"/>
+                           <material key="leather"/>
+                           <material key="textile"/>
+                        </decoNote>
+                        <decoNote xml:id="b4" type="SlipCase"> The manuscript is kept in a māḫdar.</decoNote>
+                        <decoNote xml:id="b5" type="SewingStations">4</decoNote>
+                     </binding>    
+                  </bindingDesc>
+               </physDesc>
+               
+               <history>
+                  <origin>
+                     The manuscript can be dated based on the owner information to the
+                     <origDate notBefore="1940" notAfter="1970" evidence="context">
+                        mid to late twentieth century</origDate>.
+                     No colophon notes are present in the manuscript, but the current owner<persName ref="PPRS14050MentasnotAbbuhay"/>
+                     confirmed it as the work of his father <persName ref="PRS14379Abbuhay" role="scribe"/>.                    
+                  </origin>
+                  <acquisition>later owned by the scribe<persName ref="PPRS14050MentasnotAbbuhay" role="owner"></persName> </acquisition>
```

Comment:
- replace `acquisition `by `provenance`, 
- add space between `scribe `and `<persName`
- `PRS`... not `PPRS`...